app/core/config.py

```
@@ -17,6 +18,48 @@ def get_required_env(var_name: str) -> str:
+    2. If CORE → return os.getenv(key, default) 
+    3. If SERVICE → check DevCycle first, then fallback to os.getenv(key, default)
+    
+    Supported types: str, int, float, bool
+    """
+    # TODO: You'll define the core vs service classification logic later
+    # For now, assume all are service (you'll define this later)
+    is_core = False  # We'll define this logic later
+    
+    if is_core:
+        # CORE: Direct environment variable access (like ENABLE_AIC_FILTER)
+        if type == bool:
```

Ruff (0.14.3)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)